sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -439,6 +444,7 @@ These code samples show common scenario operations with the Azure Form Recognize
 * Recognize receipts: [sample_recognize_receipts.py][sample_recognize_receipts]
 * Recognize receipts from a URL: [sample_recognize_receipts_from_url.py][sample_recognize_receipts_from_url]
 * Recognize business cards: [sample_recognize_business_cards.py][sample_recognize_business_cards]
+* Recognize invoices: [sample_recognize_invoices.py][sample_recognize_invoices]
```

Comment:
Was it intentional not including Invoices as part of the `Examples` section?
I agree with what @samvaity said once about re-thinking how to show all the prebuilt examples here, but for this first preview I think it is ok if we just list them all

Reply:
Yes, it was intentional because of how long the readme was getting. I'm fine with adding it for this preview until we have a better solution.

Reply:
yeah :( so long. I am open to other ideas too if we have time for preview.1 :)